### PR TITLE
Add a Mailto link in the search results page

### LINKED ISSUE #1194: PeopleSearch: Mailto Link in Results View
It might be helpful for the email address in the search results list to be clickable (that is, a "mailto:" link).  This would save clicking the line and then clicking the email address in the profile page.

### PR DESCRIPTION
**Pull request by charlielee144r**
Implements a use case where you can easily start emailing someone from the people search results list by adding a mailing icon/button for each result listed. 
This is only implemented in Desktop view for now because of the limited space for such a use case on a mobile view.

Fixes #1194

### CODE REVIEW

**Comment by charlielee144r** [comment]
Here is what the current styling looks like:
image

**Comment by bennettforkner** [comment]
Hi @charlielee144r , looking good so far! On the design side, I would recommend giving a little bit more left & right padding to the button as well as decreasing the border-radius.

**Comment by charlielee144r** [comment]
Current status of screen with all edits:
image

File: src/views/PeopleSearch/components/PeopleSearchResult/index.tsx [hunk 1]
```diff
@@ -81,42 +88,72 @@ const PeopleSearchResult = ({ person, lazyLoadAvatar }: Props) => {
...
+              color: gordonColors.neutral.grayShades[50],
+              height: '100%',
+              width: 40,
+              borderRadius: 2,
+            }}
+          />
+        </a>
+      </CardActionArea>
+    </div>
+  );
```
Comment (bennettforkner) [inline src/views/PeopleSearch/components/PeopleSearchResult/index.tsx:108]: You can actually set this to null if it is mobile view. A nice shortcut for specifying "only show this thing if this other thing is true" is:
```suggestion
  const emailIcon = !isMobileView && (
    <div className={styles.mailingIconContainer}>
      <CardActionArea className={styles.mailCardAction}>
        <a href={`mailto:${person.Email}`}>
          <MailOutlineIcon
            sx={{
              color: gordonColors.neutral.grayShades[50],
              height: '100%',
              width: 40,
              borderRadius: 2,
            }}
          />
        </a>
      </CardActionArea>
    </div>
  );
```

File: src/views/PeopleSearch/components/PeopleSearchResult/PeopleSearchResult.module.scss [hunk 2]
```diff
@@ -1,5 +1,11 @@
 @import '../../../../vars';
 
+.resultContainer {
```
Comment (bennettforkner) [inline src/views/PeopleSearch/components/PeopleSearchResult/PeopleSearchResult.module.scss:3]:
```suggestion
.result_container {
```
Also here (see below comment)

**Review by bennettforkner — CHANGES_REQUESTED** [review]
Good job on this Charlie, it's a great feature to add here.

I have a few small code quality suggestions in my comments.

**Review by bennettforkner — APPROVED** [review]
LGTM!

**Pull request merged**